src/socketgateway.cc

```diff
@@ -70,15 +71,16 @@ void SocketGateway::commandsWorker(int socket, std::shared_ptr<command::Channel>
 
 void SocketGateway::eventsServer() {
   events_socket = socket(AF_UNIX, SOCK_STREAM, 0);
-  struct sockaddr_un cli_addr, addr;
+  struct sockaddr_un cli_addr {
+  }, addr{};
```

Comment:
This is a typo, right? Should be all on one line.

Reply:
Yes, it is better  to do it in one line